libs/api/domains/education/src/lib/graphql/sendTeachingLicense.input.ts

```diff
@@ -0,0 +1,14 @@
+import { Allow } from 'class-validator'
```

Comment:
Don't need class validators on graphql types.

Reply:
I need it, if I skip it I get a validation error

Reply:
WTF